package.json

```diff
@@ -4,11 +4,44 @@
   "description": "facial recognition",
   "main": "app.js",
   "dependencies": {
-    "express": "^4.16.2"
+    "aws-sdk": "^2.249.1",
+    "babel-polyfill": "^6.26.0",
+    "body-parser": "^1.18.2",
+    "crypto-js": "^3.1.9-1",
```

Comment:
This library seems to have little activity over the last few years. Consider an alternative and check other packages.